task_1/sergii.garanenko/src/main/java/services/CommandService.java

```diff
@@ -0,0 +1,65 @@
+package services;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+public class CommandService {
+
+  @Autowired
+  private VersionControlService controlService;
+
+  public void listen() {
+    BufferedReader reader = new BufferedReader(new InputStreamReader(System.in, StandardCharsets.UTF_8));
```

Comment:
I might want to get to know about that library: https://commons.apache.org/proper/commons-cli/ for working with command line.